### PR TITLE
Update libyang and rtrlib

### PR DESCRIPTION
**Pull request by majst01**
Update libyang from 0.16 -> 1.0.184 as described in https://github.com/FRRouting/frr/blob/master/doc/developer/building-libyang.rst
Also update rtrlib from 0.6.3 -> 0.7.0

Create new image tags for the same frr version to be able to specify the older builds also.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,22 +6,22 @@ FROM ${OS_NAME}:${OS_VERSION} as frr-builder
 ARG FRR_TAG
 ARG RTR_TAG
 ARG LIBYANG_VERSION
-ARG LIBYANG_BUILD
 ARG LIBYANG_BUILD_ID
+ARG LIBYANG_DISTRIBUTION
 
-ENV LIBYANG_URL=https://ci1.netdef.org/artifact/LIBYANG-YANGRELEASE/shared \
+ENV LIBYANG_URL=https://ci1.netdef.org/browse/LIBYANG-LY1REL-4/artifact/shared/${LIBYANG_DISTRIBUTION}-x86_64-Packages \
```
Comment (mwindower) [inline Dockerfile]: Would be clearer if `4` is externalised as own ENV because it is the bamboo CI-Build number of libyang

Comment (majst01) [reply Dockerfile]: Yes will do

Comment (majst01) [reply Dockerfile]: Problem is which version to choose:
https://ci1.netdef.org/browse/LIBYANG-LY1REL

Comment (majst01) [reply Dockerfile]: took the latest build now

**Review by mwindower — APPROVED** [review]
LGTM

**Pull request merged**